event_parsing.go

```diff
@@ -252,6 +253,20 @@ func ParseWebhook(eventType EventType, payload []byte) (event interface{}, err e
 		event = &PushEvent{}
 	case EventTypeRelease:
 		event = &ReleaseEvent{}
+	case EventTypeResourceAccessToken:
+		data := map[string]interface{}{}
+		err := json.Unmarshal(payload, &data)
+		if err != nil {
+			return nil, err
+		}
+
+		if _, ok := data["project"]; ok {
```

Comment:
The payload is mostly the same for both types. Depending on the actual type, either `group_id`/`group` or `project_id`/`project` will be present. A bit of a change compared to other events… 🤷